src/us/kbase/workspace/test/workspace/ObjectInformationTest.java

```diff
@@ -185,6 +189,40 @@ private void buildFullWithAlternativeInputsSharedChecks(final ObjectInformation
 				is(Arrays.asList(new Reference(1, 67, 24))));
 	}
 	
+	@Test
+	public void buildFromObjectInfoMinimal() throws Exception {
+		final ObjectInformation source = minimalOI();
+		final ObjectInformation target = ObjectInformation.getBuilder(source).build();
+
+		assertThat("incorrect build", target, is(source));
+	}
+	
+	@Test
+	public void buildFromObjectInfoMaximal() throws Exception {
+		final ObjectInformation source = buildFullWithAlternativeInputsSetup()
+				.withUserMetadata(new UncheckedUserMetadata(ImmutableMap.of("a", "b")))
+				.withAdminUserMetadata(new UncheckedUserMetadata(ImmutableMap.of("e", "f")))
```

Comment:
I had a small cognitive hiccup seeing 
```java
    .withUserMetadata(new UncheckedUserMetadata(...))
    .withAdminUserMetadata(new UncheckedUserMetadata(...))
```
because at first my brain was expecting to see `new UncheckedAdminUserMetadata` as input for the second method. Would the class name `UncheckedMetadata` would be too ambiguous? Hmmm.

The hiccup resolved itself very quickly so unless you had any naming doubts, it's probably not worth changing.